### DISCUSSION TITLE
Hi I have a problem during installation

### DISCUSSION THREAD

**Discussion by ekfldntm** [Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh#47] [Install Help]
I'm using ux481F model and I tried hackintosh with ventura(13.5) with your efi. I did all of the steps in your instructions. However, during installation process, every time(I tried about 10 times lol) when screen shows "about 12 minutes remaining", laptop stops. And any keys don't work. Is it the problem of my model or are there any other any problems? plz helps me

Here is my efi(I ran build.sh to build efi and after that, I didn't change anything)
[File](https://jbmulb.duckdns.org:25001/d/s/unWtbFApNfpKpfYoKjZyBMmZaAOTCR68/39U8OHYrJgOOp2q7cy_9mjSDfz4Aa-66-YLhAE6Omqwo) - if website says "Your connection is not private - join

**Comment by Qonfused** [comment]
I've noticed a similar installer bug on Ventura 13.4 that could be related (https://github.com/Qonfused/OSX-Hyper-V/issues/1). The workaround I found was to upgrade from Monterey (basically just download Monterey and work from there).

In your case, I'm not sure if this issue could persist if you needed to use macOS Recovery later. There may be more clues about what is happening in the installer log that could help. You can find this under `Window > Installer Log` and can also save it to disk:

![image](https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/assets/32466081/fcf95510-0e0b-4418-a2d2-db56a375c1b5)
![image](https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/assets/32466081/ba72e003-0a8c-4df8-a698-0faa6cf60c33)

**Reply by ekfldntm** [reply]
I tried it on Monterey, but same problem occur. Laptop stops at "about 12 minutes remaining".
![1](https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/assets/78641747/246d8839-c704-4c77-955b-056583a015c0)
After stop, I can't do anything except turning off


I also attach log file
[Installer Log 14-Aug-2023-1.txt](https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/files/12340455/Installer.Log.14-Aug-2023-1.txt)
```
04a1a000 : Failed to create systemVersionDict
04a1a000 : Failed to get currentInstalledBuild. Assuming clean install
04a1a000 : Failed to create systemVersionDict
04a1a000 : Failed to get currentInstalledBuild
```

**Reply by Qonfused** [reply]
I'm assuming `disk5s1` is the disk you're installing macOS on?

```
Aug 15 01:00:19 MacBook-Pro osinstallersetupd[185]: isConvertableToAPFS: was called on disk5s1, an APFS disk.
Aug 15 01:00:19 MacBook-Pro osinstallersetupd[185]: disk5s1 is not convertible to APFS: This volume is not formatted as Mac OS Extended (Journaled).
```

There the installer is expecting an HFS partition rather than an APFS partition for the install target. About the 12-minute mark is when most disk writing starts taking place, and here it can sometimes fail when targeting an APFS disk without an existing macOS install.

I'd recommend formatting that partition as `Mac OS Extended (Journaled)` (aka HFS Journaled) in Disk Utility. This will automatically be converted to APFS when installing macOS.

**Reply by Qonfused** [reply]
It may be expecting to restart as well, but the abrupt halt would seem to indicate some kind of kernel panic or resource conflict. I'd also double check you've configured your BIOS's SATA and Fast Boot settings to be sure: [Install-Guide-(WIP)](https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/wiki/Install-Guide-(WIP)).

**Reply by ekfldntm** [reply]
I formatted the partition as `Mac OS Extended (Journaled)`,and I finally escaped from the 12-minute mark. However, after it passes 12 minute mark, I met another kernel panic. After the panic, I can't boot computer again so I couldn't get log file.

https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/assets/78641747/0a5d3faf-01d2-45b5-92e8-186769948b29

My bios settings
```
Advanced :
- Internal Pointing Device : Enabled
- Fn lock Option : Enabled
- Hyper-Threading : Enabled
- Intel Virtualization Technology: Enabled
- Intel AES-NI : Enabled
- VT-d : Enabled
- SMART Self Test : Enabled
- Network Stack : Disabled
- USB Mass Storage Driver Support : Enabled
- DVMT Pre-Allocated : 64M
- SATA Mode Selection : AHCI

Boot :
- Fast Boot : Disabled

Security :
- I/O Interface Security : everything UnLock
- Secure Boot Control : Disabled
```

**Reply by Qonfused** [reply]
Seems to be caused by some kind of conflict with the NVMe driver:

![image](https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/assets/32466081/f6cf74a4-2199-4d96-ad41-0839e6cc4934)

Do you know what SSD you're using with this laptop?

**Reply by ekfldntm** [reply]
I've never changed ssd after I bought this laptop. To know exact ssd, I have to install other OS. Yet, It's currently night in my country, so I might try it tomorrow. (I live in Korea (time : GMT+9), can you tell me what the time difference is from here?)

**Reply by Qonfused** [reply]
The stock SSD should be the Intel 660p, which I've tested to verify it's supported. It's 9 am here (UTC-5), so I'm 14 hours behind.

**Reply by ekfldntm** [reply]
I checked my SSD. It is "NVMe SAMSUNG MZVLBITO" 1Tb.
![1](https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/assets/78641747/b06ce588-9997-4c51-81b7-1a378325f16e)

**Reply by ekfldntm** [reply]
I passed the 12-minute mark. And, I finally reach to this screen
![IMG_0031](https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/assets/78641747/7245995b-c989-4fd0-863c-a7641f237327)
However, after I click right icon : macOS installer, and then boot loop is occur. (but no panic)

https://github.com/Qonfused/ASUS-ZenBook-Duo-14-UX481-Hackintosh/assets/78641747/1088cf98-fff9-4404-8f38-094bd67075bc

**Reply by Qonfused** [reply]
I'd try the 'Install macOS Monterey' option there; the 'macOS Installer' option might be pointing to the APFS macOS install being created, but it's hard to tell from the video (and frankly I'm not familiar with the handoff process with the installer).

When running the installer normally, there may be a few reboots needed where you'd have to manually select and boot into the installer again. Usually, you'll see a 'macOS' partition rather than an 'Install macOS ...' or 'macOS Installer' partition in the OpenCore boot picker once it has finished installation.

**Reply by Qonfused** [reply]
Another thing, the 'Samsung MZVLB1T0HALR' drive appears to be a Samsung PM981 SSD which unfortunately isn't supported by macOS. This conflicts with the IONVMeController as you saw before in the installer log, which may also cause kernel panics when rebooting into the installer.

You'll have to replace this drive as macOS will refuse to work with it at all. You can try your luck with it but most likely you'll end up with a kernel panic pointing to a IONVMeController timeout.
